web/src/main/webapp/WEB-INF/jsp/managestudy/viewStudySubject.jsp

```diff
@@ -2,58 +2,69 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
```

Comment:
Like to talk about these changes before merging it in

Reply:
fixed again in commit eb20d8d27658bb4c29bd8008da7fc985820ca68c